alisw/AliRoot
# ATO-418, ATO-445 - Adding tutorials + AliPainter,CSS updates  + Doxygen

# ATO-418: AliPainter and AliDrawStyle (Boris)
##  AliPainter
* Warning output for PandasParser was simplified.
##  AliDrawStyle:
* Bug with margins(if specified side-margin it was applied to all margin) fixed;
* Added ndivision property(ndivisions for both -x,y and x-ndivisions, y-ndivisions);
* Function ReadCSSString was added. Now you can copy content of your css file to TString and parse it to array;
* List of available properties was added to the ApplyCssStyle description;
Code was installed without error. All tests passed.

 
# Tutorials - Create Notebooks directory  (Marian)
* TPCDataVolume.C macro with parallel  TPCDataVolume.ipynb
* Add Notebooks to test and tutorials
* add test to default tests
* extend doxygen documentation - including generatig figures
  * including log output from macro does not work
* check with root team setup for the macro
````
///\macro_output
///\macro_code
